lua/autorun/sh_portal_movement.lua

```diff
@@ -116,6 +127,8 @@ local function editPlayerCollision(mv, ply)
 				return -- we accomplished nothing :DDDD
 			end
 
+			-- What the fuck is this?
+			-- TODO: Comment what this does
```

Comment:
if the portal is directly facing up, then set the player collision hull offset to 72 HU